data/json/recipes/weapon/cutting.json

```diff
@@ -141,6 +142,21 @@
     "proficiencies": [ { "proficiency": "prof_carving", "time_multiplier": 1.5, "fail_multiplier": 1 } ],
     "components": [ [ [ "duct_tape", 100 ] ], [ [ "blade", 1 ] ], [ [ "stick_long", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "makeshift_glaive",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "45 m",
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 }, { "id": "DRILL", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_carving", "time_multiplier": 1.5, "fail_multiplier": 1.5 } ],
+    "components": [ [ [ "duct_tape", 100 ] ], [ [ "blade", 1 ] ], [ [ "stick_long", 1 ] ], [ [ "cordage", 2, "LIST" ] ] ]
```

Comment:
The description mentions "bolted", and I see HAMMER and DRILL, but what's being used for "bolts"?

Reply:
I'm not a 100% sure myself, i'm copying directly from the same use of language/tools as the "better" knife spear and just assumed that was correct, it's very possible both items should be adjusted.

Reply:
One other thing: Just "blade" and not the other two you removed from the knife spear recipe?

> I'm not a 100% sure myself, i'm copying directly from the same use of language/tools as the "better" knife spear and just assumed that was correct, it's very possible both items should be adjusted.

Yes.

> Some very cursory searching says that "bolt ropes" (bolting in place with rope) is a thing and i assume that is what is meant.

Ah; passing the rope through a drilled hole in the handle? A weird/unclear way to say it, IMO, but OK.

Reply:
I thought the 2 cleavers could be disassembled to get the "blade" item but that does not seem to be the case, which in retrospect seems fair, since the "blade" item is heavier and bigger on its own than both knives. 

The cleavers are a bit small for making into a polearm, though there is an argument to be made for adding them in as alternatives to the blade. I will leave them out for now and see what others think.

Reply:
> I thought the 2 cleavers could be disassembled to get the "blade" item but that does not seem to be the case, which in retrospect seems fair, since the "blade" item is heavier and bigger on its own than both knives.

Are you quite sure on that? Or is items/recipe_deconstruction.json even, umm, less clear than it initially appears?

> The cleavers are a bit small for making into a polearm, though there is an argument to be made for adding them in as alternatives to the blade. I will leave them out for now and see what others think.

I'm reasonably certain that the original glaive was constructed by taking a butcher's cleaver and strapping it to the end of a pole... it does depend on size, true.

Reply:
You are entirely correct, i was looking at stable chezzo rather than experimental, cleavers deconstruct into blades now. I'll add the cleavers as alternate items to make it more obvious to the player they are usable. 
(because they can now be used without first disassembling.)